src/axios-har-tracker.ts

```diff
@@ -96,17 +92,42 @@ async function axiosTracker(call) {
             _blocked_queueing: -1
           }
         };
-        const enteriesContent = Object.assign({}, newEntry);
-        generatedHar.log.entries.push(enteriesContent);
+        this.enteriesContent = Object.assign({}, this.newEntry);
```

Comment:
I still think that this is not needed, in fact the whole entriesContent is not needed. I think that this will be enough
```js
this.generatedHar.log.entries.push(this.newEntry);
```

Reply:
without this, there is a problem with collecting the content of the object - in short word without this newEntry in just a null